service/service.go

```diff
@@ -357,25 +358,48 @@ func (c *ComplianceScanService) loopRegister() {
 	}
 }
 
+func (c *ComplianceScanService) queryAndRegisterCloudResourcesPeriodically() {
+	refreshTicker := time.NewTicker(12 * time.Hour)
```

Comment:
can we use some kind of cron here instead of long running timer ? what if cloud scanner restarts ?